core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1985,15 +1990,15 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
 
             val defaultPolicies = Map(projectOwnerPolicy, ownerPolicy, writerPolicy, readerPolicy, shareReaderPolicy, shareWriterPolicy, canComputePolicy, canCatalogPolicy)
 
-            DBIO.from(samDAO.createResourceFull(SamResourceTypeNames.workspace, workspaceId, defaultPolicies, workspaceRequest.authorizationDomain.getOrElse(Set.empty).map(_.membersGroupName.value), userInfo))
+            traceDBIOWithParent("createResourceFull", parentSpan)(_ => DBIO.from(samDAO.createResourceFull(SamResourceTypeNames.workspace, workspaceId, defaultPolicies, workspaceRequest.authorizationDomain.getOrElse(Set.empty).map(_.membersGroupName.value), userInfo)))
```

Comment:
It seems a little weird to me put Sam calls inside a DB transaction. That is not new in your PR, just commenting.

Reply:
I wonder if it would be better to do
```
DBIO.from(traceWithParent("createResourceFull", parentSpan)(_ => samDAO.createResourceFull(…))
```
since this _really_ isn't a database call..

Reply:
sam calls inside a db transaction is indeed one of the Bad Things we do, ticketed on app services as [AS-9](https://broadworkbench.atlassian.net/browse/AS-9)